WickedEngine/wiSDLInput.cpp

```diff
@@ -80,18 +80,6 @@ namespace wiSDLInput
 
                         // mouse events
                     case SDL_MOUSEMOTION:          // Mouse moved
-                        mouse.position.x = event.motion.x;
-                        mouse.position.y = event.motion.y;
-                        mouse.delta_position.x += event.motion.xrel;
-                        mouse.delta_position.y += event.motion.yrel;
-                        // TODO you can update mouse buttons by reading event.motion.state
-                        mouse.left_button_press = event.motion.state & SDL_BUTTON_LMASK;
-                        mouse.right_button_press = event.motion.state & SDL_BUTTON_RMASK;
-                        mouse.middle_button_press = event.motion.state & SDL_BUTTON_MMASK;
-                        //mouse.x1_button_press = event.motion.state & SDL_BUTTON_X1MASK;
-                        //mouse.x2_button_press = event.motion.state & SDL_BUTTON_X2MASK;
-
-                        break;
```

Comment:
I think we should not remove this `break;`

Reply:
Just wanted to be consistent with the source file. Because all other events are "grouped" and have one break per group.

But I can insert it, doesn't change the behaviour anyways.